js/ui/drop_down_editor/ui.drop_down_list.js

```diff
@@ -329,6 +329,14 @@ var DropDownList = DropDownEditor.inherit({
         this._setListDataSource();
         this._validateSearchMode();
         this._clearSelectedItem();
+        this._initItems();
```

Comment:
Use `this._refreshDataSource();`

Reply:
This method does not suit us, because happens re-init data source and a loads data on the init of SelectBox.